Makefile

```diff
@@ -68,6 +68,8 @@ else
 ALL_PACKAGES_COMMA=$(TE_PACKAGES_COMMA)
 endif
 
+default: build-current
```

Comment:
I thought we might have wanted the `make all` for whatever reason, but I can just use that